docs/DeveloperGuide.md

```diff
@@ -397,12 +397,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user                                       | see the profile of a particular student           | get their information                                        |
 | `* * *`  | user                                       | remove a student's profile                        | keep track of only students that I teach                     |
 | `* * *`  | user                                       | edit the details of a student                     | keep track of up-to-date information                         |
-| `* * *`  | user                                       | add individual tuition sessions                   | keep track of my tuition sessions                            |
-| `* * *`  | user                                       | delete an individual tuition session              | update my tuition schedule                                   |
+| `* * *`  | busy user                                  | add an individual tuition sessions                | save time by creating multiple sessions in one command       |
```

Comment:
sorry ah, what's this referring to? 'save time by creating multiple sessions', are you referring to recurring?

Reply:
Yeap sorry man! The "save time by creating multiple sessions" is supposed to be for recurring session, have updated it accordingly.